app/script/setup-flutter.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+if [ "$GAE_SERVICE" != "analyzer" ];
+then
+  echo "Flutter setup aborted: GAE_SERVICE must be \"analyzer\"."
+  exit 0
+fi
+
+if [[ -z "$FLUTTER_SDK" ]];
+then
+  echo "Flutter setup aborted: FLUTTER_SDK must point to a target directory."
+  exit 0
+fi
+
+if [ -d "$FLUTTER_SDK" ];
+then
+  echo "Flutter setup aborted: $FLUTTER_SDK already exists, assuming proper setup."
+  # This shouldn't happen in docker, making sure this fails during an image build.
+  exit 1
+fi
+
+git clone -b alpha https://github.com/flutter/flutter.git $FLUTTER_SDK
```

Comment:
Could we use `set -e` in the top of this file, to ensure a `git clone` failure (or any other failure in this shell script) is fatal?